hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/statemachine/DatanodeConfiguration.java

```diff
@@ -89,10 +89,10 @@ public class DatanodeConfiguration {
 
   public static final String
       OZONE_DATANODE_CHECK_EMPTY_CONTAINER_ON_DISK_ON_DELETE =
-      "hdds.datanode.check.empty.container.delete";
+      "hdds.datanode.check.empty.container.on-disk.on.delete";
```

Comment:
Minor suggestion since technically the RocksDB is still being checked and that is on the disk too.
```suggestion
      "hdds.datanode.check.empty.container.dir.on.delete";
```

Reply:
Also we should probably add an `@Config` field with a description to make this consistent with the other configs in this class.

Reply:
Updated config and description.